Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -229,9 +229,9 @@
         - CigPack
         - Pill
         - PillCanister
+        - Radio
```

Comment:
Security belts should probably be able to hold them too if medical belts can.

Reply:
Sure, ill add that. Whether or not belts can hold radios seems to be very arbitrary and inconsistent.